packages/terra-form/src/Control.scss

```diff
@@ -1,22 +1,26 @@
 @import '~terra-mixins';
```

Comment:
As these components have been updated to use css modules, terra-mixins have been replaced with the defaulted `dir=ltr` behavior since the `postcss-rtl` webpack plugin can handle the directionality.